src/frontend/src/components/CountingCard/styles.module.scss

```
@@ -0,0 +1,38 @@
+@use "../../styles/variables" as v;
+
+.container {
+  background-image: url(/background-small-card-3.png);
+  width: 327px;
+  height: 365px;
+  background-size: cover;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.container__text {
+  text-align: center;
+  font-family: "Drive Corps", sans-serif;
```

Agora, dentro do `_variables`, tem a variável da família de font "DriveCorps". Então, realizar essa alteração:
```suggestion
  font-family: v.$DriveCorps;
```